include/openspace/util/touch.h

```diff
@@ -72,12 +73,14 @@ class TouchInputHolder {
     double gestureTime() const;
 
     size_t numInputs() const;
+    const TouchInput& firstInput() const;
```

Comment:
Why is it needed to keep track of the firstInput?

Reply:
For continuous touch I basically measure the distance between the current distance from the fingers to their centroid point against the distance from the first ever pinch-input against the first ever centroid between those two fingers, if that makes sense.

The difference between current and first distance from centroid is what gives the current zoom velocity.

Reply:
Ok so the speed of the initial pinch is the speed that will be held?

Reply:
Yeah, basically:
- initial touch with 2 fingers: save the location of these two fingers and give 0 zoom-speed since we haven't moved
- move out from center with both fingers: increase zoom speed outwards based on finger-distance travelled from initial touch
- move back to initial touch location: decrease zoom once more to 0 speed
- move in towards center with both fingers: increase zoom speed inwards based on finger-distance travelled from intitial touch

In the future I would like to use finger_distance_travelled in millimeters or such, so that we can have some scaling factor based on display size that controls all of this.